src/collator.py

```diff
@@ -36,19 +47,33 @@ def prepare_sop_from_examples(self, examples: List[Dict[str, Any]]) -> List[Dict
             reverse = random.random() < 0.5
 
             if reverse:
-                token_a, token_b = chunk_ids[split_position:], chunk_ids[:split_position]
+                token_a, token_b = (
+                    chunk_ids[split_position:],
+                    chunk_ids[:split_position],
+                )
```

Comment:
```suggestion
                token_a = chunk_ids[split_position:]
                token_b = chunk_ids[:split_position]
```

뭔가 black이 강제한 듯 하군요. 제가 보기엔 이게 좀 더 직관적일 듯 합니다~